packages/story-editor/src/components/floatingMenu/elements/shared/useFlip.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import useProperties from './useProperties';
+import useUpdateSelected from './useUpdateSelected';
+
+function useFlip() {
+  const { flip } = useProperties(['flip']);
+  const updateSelected = useUpdateSelected();
```

Comment:
as far as perf goes for this PR, this seems like the two key areas that would effect it and both `useUpdateSelected` & `useProperties` look pretty lean as far as using `useContextSelector` well. nice!